### ISSUE TITLE
Accessibility issue: Video autoplays

### ISSUE THREAD

**Issue by JenM00re** [jerry-604/swarthmore-a11ygator#2]
Accessibility issue with videos: should not / cannot autoplay I think this is the documentation: [Understanding Success Criterion 2.3.3: Animation from Interactions | WAI | W3C](https://www.w3.org/WAI/WCAG21/Understanding/animation-from-interactions.html)]

**Comment by JenM00re** [comment]
Tony suggested the media css prefers reduced motion. 

So, it'll be similar to the code from the other issue:
```
@media (prefers-color-scheme: dark) {
body { /* Light text on dark background */
color: #ffffff;
background-color: #222222;
}
}
```
